Add section describing changing the agent service user on Windows (#1890)
buildkite/docs: pages/agent/v3/windows.md

nssm status buildkite-agent
```

If you'd like to change the user the buildkite-agent service runs as, you can use the same third-party tool [nssm](https://nssm.cc/) using the command-line:

```
# Change ComputerName to the name of the System under "Settings"
# Change AccountName to the name of the Account you'd like to use

> # Change password to the password of the account you provided in the above comment

Reviewer: "The above comment" is a bit ambiguous. Perhaps:

```suggestion
# Change password to the password of the account you'd like to use
```

What do you think?

Author: 100%, great change

Author: But, keep in mind the password has to be the password already determined for the account the user is wanting to use. So if the Admin account is PC\Administrator with a password of "notarealpassword", the password field in the CLI command has to be "notarealpassword" or some kind of variable/env var to match the account password. It's not meant to be something to set the password itself. So we might wanna reconsider the wording here again.

Reviewer: I suspect the formatting is working against us here. I'm going to suggest a different style.

> nssm set buildkite-agent ObjectName "ComputerName\AccountName" "password"

Reviewer: What's the `ObjectName`? It looks like it's not mentioned anywhere here. Is it meant to be replaced by another value or left as is?

Author: So in this case, `ObjectName` is part of the CLI syntax and not to be touched.

> ```
> # Change ComputerName to the name of the System under "Settings"
> # Change AccountName to the name of the Account you'd like to use
> # Change password to the password of the account you provided in the above comment
> nssm set buildkite-agent ObjectName "ComputerName\AccountName" "password"
> ```

Reviewer: Alternative style:

````suggestion
```
nssm set buildkite-agent ObjectName "COMPUTER_NAME\ACCOUNT_NAME" "PASSWORD"
```

Replace the following:

- `COMPUTER_NAME`: The system name under _Settings_. For example, `PC`.
- `ACCOUNT_NAME`: The name of the account you'd like to use. For example, `Administrator`.
- `PASSWORD`: The password for the account you'd like to use. You can reference a variable rather than directly specifying the value.
````

This:

- Makes the command able to be copied without the comments getting in the way.
- Separates placeholders from keywords following the [Google style guide](https://developers.google.com/style/code-syntax).
- Removes ambiguity about "changing" things.

What do you think?

Author: Much nicer! I'm all for this change.
